docs/application/dotnet/index.md

```diff
@@ -1,73 +1,137 @@
-# Tizen .NET Application
+# .NET Application
```

Comment:
@mijin85cho, the UI of this page on staging does not look good, however would this appear differently or correctly when merged?

Reply:
As the test URL is shared, I think we can suppose the view in that test URL would be the merging result including other updates.

With that assumption, the view would be like below
![image](https://user-images.githubusercontent.com/9958374/114152635-76d8f080-9959-11eb-982b-f71084bfaf37.png)

I hope it helps your checking.